pkg/sources/transformer/grpc_transformer.go

```diff
@@ -127,3 +127,7 @@ func (u *gRPCBasedTransformer) ApplyMap(ctx context.Context, readMessage *isb.Re
 	}
 	return taggedMessages, nil
 }
+
+func (u *gRPCBasedTransformer) ApplyMapStream(_ context.Context, _ *isb.ReadMessage, _ chan<- isb.WriteMessage) error {
+	return fmt.Errorf("method ApplyMapStream not implemented")
```

Comment:
I don't think this matters, since this is not supported and we won't even allow transformer type with forwarder to be initialized. See [here](https://github.com/numaproj/numaflow/pull/709/files#diff-72611cd85ccc44e7cd2fcc37a91e5ac5c4973178be377d0a00ff41f65aee25acR116)